cabal-install/Distribution/Client/SetupWrapper.hs

```diff
@@ -365,7 +365,7 @@ runSetup verbosity setup args0 = do
 -- verbosity applies to ALL commands.
 verbosityHack :: Version -> [String] -> [String]
 verbosityHack ver args0
-    | ver >= mkVersion [1,25] = args0
+    | ver >= mkVersion [2,1]  = args0
```

Comment:
@23Skidoo now that you mention it... yes that makes sense... I wonder how I can trigger that code-path... is that old-build?